### PR TITLE
Update homework.sh

### PR DESCRIPTION
**Pull request by IhorKyl**
homework shell session
task 1-10 done

## @mentions of the person or team responsible for reviewing proposed changes (At least 2 people
@dtxe
@michaeladrouillard

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 03_homework/homework.sh [hunk 1]
```diff
@@ -7,14 +7,30 @@
...
+ls fld1/*.txt
+ls fld2/*.txt
+ls fld3/*.txt
+ls fld4/*.txt
+ls fld5/*.txt
 
 # 5. How would you append to one of the existing files " and machine learning!"?
+echo " and machine learning!" >> fld1/f.txt
 
 # 6. How would you verify that the text was indeed appended to the existing file?
```
Comment (dwz92) [inline 03_homework/homework.sh:35]: Missing Q6 to Q10

Comment (IhorKyl) [reply 03_homework/homework.sh:35]: > Missing Q6 to Q10

Hi. I did second commit with "Q6-Q10". 
Can you see it?

**Review by juliagallucci — APPROVED** [review]
Good!

**Review by juliagallucci — APPROVED** [review]

**Comment by IhorKyl** [comment]
uoft-dsi-c3-homework-20240426

**Pull request closed without merging**